pkg/manager/member/template.go

```diff
@@ -341,11 +341,19 @@ var tidbInitStartScriptTpl = template.Must(template.New("tidb-init-start-script"
 host = '{{ .ClusterName }}-tidb'
 permit_host = '{{ .PermitHost }}'
 port = 4000
+for i in range(0, 10):
+    while True:
```

Comment:
Need to use both `for` and `while` here?

Reply:
if after 10 times try and all failed it will continue to run, but `conn` is not connect successful ?
i think this is not expected